README.md

```diff
@@ -103,11 +103,12 @@ Citations are beneficial for the growth and improvement of mlpack.
 
 ## 2. Dependencies
 
-mlpack requires a C++14 compiler and has the following additional dependencies:
-
- - Armadillo      >= 9.800
- - ensmallen      >= 2.10.0
- - cereal         >= 1.1.2
+**mlpack** requires the following additional dependencies:
+ - C++14 Compiler
+ - [Armadillo](https://arma.sourceforge.net/docs.html)      &nbsp;&emsp;>= 9.800
```

Comment:
Why point to docs.html ?

Please use the main site: https://arma.sourceforge.net

Reply:
I don't know why I did that but will change to the main site.